src/librustdoc/html/static/main.js

```diff
@@ -1623,6 +1623,7 @@
     function toggleAllDocs() {
         var toggle = document.getElementById("toggle-all-docs");
         if (hasClass(toggle, "will-expand")) {
+            updateLocalStorage("collapse", "false");
```

Comment:
It seems like "too simple" for a name. Can you replace it with "rustdoc-collapse" please?

Reply:
Also, please use booleans instead of strings.

Reply:
The values end up in localStorage, which [converts all values to strings](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API) so I thought it clearer to avoid any type coercion and use strings always.

Reply:
Oh I didn't know, thanks for the clarification! Then just ignore this comment.